lib/coinatmradar/coinatmradar.js

```diff
@@ -147,3 +151,7 @@ function update (info) {
     .then(sendRadar)
     .catch(err => logger.error(err))
 }
+
+function computeOperatorId (masterSeed) {
+  return hkdf(masterSeed, 32, {salt: 'lamassu-server-salt', info: 'operator-id'}).toString('hex')
+}
```

Comment:
Configure your editor to require blank lines at EOF.

Reply:
Oops, I thought that was up. I'm going to configure stardand to run on npm test while adding the elm-admin to l-s.